ui/listner-api/src/main/scala/pl/touk/nussknacker/listner/ChangeEvent.scala

```diff
@@ -0,0 +1,17 @@
+package pl.touk.nussknacker.listner
+
+import pl.touk.nussknacker.engine.api.process.ProcessName
+
+sealed trait ChangeEvent
```

Comment:
maybe `ProcessChangeEvent` to make it visible that all events are in context of process? Also shouldn't it be processId everywhere? I would like to write listener implementation that will partition all events by some identifier to ensure that changes will have correct order for this identifier.

Reply:
I like the idea of `ProcessId` everywhere so I've added that to this trait. I have also removed `ProcessName` from some events because its unnecessary now.
Ofc youre write about name - also changed.